doc/tutorials/enable_s5.rst

```diff
@@ -97,7 +99,6 @@ The procedure for enabling S5 is specific to the particular OS:
      :name: laag-waag-script
```

Comment:
I don't thing the lines we want to highlight still appear in those scripts... @NanlinXie , can ask one of the developer to look at this?